crates/red_knot_python_semantic/resources/mdtest/stubs/class.md

```diff
@@ -6,7 +6,12 @@ In type stubs, classes can reference themselves in their base class definitions.
 `typeshed`, we have `class str(Sequence[str]): ...`.
 
 ```py path=a.pyi
```

Comment:
Did you verify that this test fails if you make this a `.py` instead of a `.pyi`? (Just want to make sure it's still testing something meaningful.)

Reply:
Yes, it still fails if it's a `.py` file!

```
    /stubs/class.md:15 unmatched assertion: revealed: Literal[Bar]
    /stubs/class.md:15 unexpected error: 1 [revealed-type] "Revealed type is `Unknown`"
    /stubs/class.md:16 unmatched assertion: revealed: tuple[Literal[Bar], Unknown, Literal[object]]
    /stubs/class.md:16 unexpected error: 1 [revealed-type] "Revealed type is `Unknown`"
```